typescript/submissionEvents.d.ts

```diff
@@ -37,6 +47,10 @@ export type PdfSubmissionEvent = FormSubmissionEventConditional & {
     includeSubmissionIdInPdf?: boolean
     excludedElementIds?: string[]
     usePagesAsBreaks?: boolean
+    emailTemplate?: {
```

Comment:
Should we also add it to other submission event types that produce PDF's? e.g. TRIM, Civica...

Reply:
This is not PDF config, its email config.